src/components/com_tjcertificate/administrator/models/forms/trainingrecord.xml

```diff
@@ -0,0 +1,132 @@
+			type="text"
+			filter="safehtml"
+			label="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_ID"
+			description="COM_TJCERTIFICATE_CERTIFICATE_FORM_DESC_CERTIFICATE_ID"
+		/>
+
+		<field
+			name="cert_url"
+			type="text"
+			label="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_URL"
+			description="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_URL_DESC"
+			menu_published="0"
+			filter=""
+			validate="url" 
+			schemes="http,https"
+		/>
+
+		<field
+			name="issuing_org"
+			type="text"
+			label="COM_TJCERTIFICATE_FORM_LBL_ISSUE_ORG"
+			description="COM_TJCERTIFICATE_FORM_LBL_ISSUE_ORG_DESC"
+			required="true"
+			menu_published="0"
+			filter=""
+		/>
+
+		<field name="comment"
+			type="textarea"
+			label="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_COMMENT"
+			description="COM_TJCERTIFICATE_CERTIFICATE_FORM_DESC_CERTIFICATE_ISSUED_COMMENT"
+			cols="45"
+			rows="10"
+		/>
+
+		<field
+			name="issued_on"
+			type="calendar"
+			required="true"
+			default=""
+			filter="user_utc"
+			format="%Y-%m-%d %H:%M:%S"
+			singleheader="true"
+			showtime="true"
+			timeformat="24"
+			todaybutton="false"
+			weeknumbers="false"
+			filltable="true"
+			label="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_ISSUED_DATE"
+			description="COM_TJCERTIFICATE_CERTIFICATE_FORM_DESC_CERTIFICATE_ISSUED_DATE"
+		/>
+
+		<field
+			name="expired_on"
+			type="calendar"
+			default=""
+			filter="user_utc"
+			format="%Y-%m-%d %H:%M:%S"
+			singleheader="true"
+			showtime="true"
+			timeformat="24"
+			todaybutton="false"
+			weeknumbers="false"
+			filltable="true"
+			label="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_EXPIRY_DATE"
+			description="COM_TJCERTIFICATE_CERTIFICATE_FORM_DESC_CERTIFICATE_EXPIRY_DATE"
+			onchange="tjCertificateService.validationEndDate(this)"
+			class="validate-expdate"
+		/>
+
+		<field name="status" type="list" label="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_STATUS" description="COM_TJCERTIFICATE_CERTIFICATE_FORM_LBL_CERTIFICATE_STATUS_DESC">
+		  <option value="attended">COM_TJCERTIFICATE_CERTIFICATE_FORM_STATUS_ATTENDED</option>
+		  <option value="completed">COM_TJCERTIFICATE_CERTIFICATE_FORM_STATUS_COMPLETED</option>
+		  <option value="passed">COM_TJCERTIFICATE_CERTIFICATE_FORM_STATUS_PASSED</option>
+		  <option value="failed">COM_TJCERTIFICATE_CERTIFICATE_FORM_STATUS_FAILED</option>
+		</field>
+
+		<field
+			name="cert_file"
+			type="file"
+			label="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_FILE"
+			description="COM_TJCERTIFICATE_FORM_LBL_CERTIFICATE_FILE_DESC"
+			onchange="tjMediaFile.validateFile(this)"
+			accept="image/jpeg,image/png,image/jpg,application/pdf" />
+
+		<field
+			name="state"
+			type="list"
+			filter="int"
+			label="JSTATUS"
+			description=""
+			default="1"
+			class="inputbox"
+			size="1">
+				<option value="1">JPUBLISHED</option>
+				<option value="0">JUNPUBLISHED</option>
+				<option value="-1">COM_TJCERTIFICATE_STATUS_PENDING</option>
```

Comment:
We don't need this `COM_TJCERTIFICATE_STATUS_PENDING` while adding from the admin side

Reply:
removed